java/README.md

```diff
@@ -29,6 +42,52 @@ To hit the url from outside the docker compose environment:
       OTEL_SERVICE_NAME: workshop-java-app
 ```
 - Rebuild and restart docker-compose, as above.
+- Run `curl  http://localhost:8080/` -- what happened?
+- Run `curl http://localhost:8080/hello` -- what happened?
+- Run `curl http://localhost:8080/hello/proxy/node/python` -- what happened?
+
+Assuming everything is configured properly, you should see the
+`workshop-java-app` service in Jaeger.
+
+## Lab 102: Send data to the OpenTelemetry Collector
+
+By default, auto-instrumentation sends data via OTLP on `localhost:4317`. In
+Lab 101 we overrode this via environment variables. Let's modify them:
+
+- In the `docker/docker-compose.yml` file, change the `environment` section of the java service:
+```
+    environment:
+      OTEL_EXPORTER_OTLP_ENDPOINT: http://otelcol:4317
+      OTEL_SERVICE_NAME: workshop-java-app
+```
+- Restart docker-compose as above.
+- Run `curl  http://localhost:8080/` -- what happened?
+- Run `curl http://localhost:8080/hello` -- what happened?
+- Run `curl http://localhost:8080/hello/proxy/node/python` -- what happened?
+
+Assuming everything is configured properly, you should see the
+`workshop-java-app` service in Jaeger.
+
+## Lab 103: Instrument another application and call it
+
+Instrument another application (another language). Follow Lab 101 for that
+other application. Then run `curl
+http://localhost:8080/hello/proxy/<otherLanguage>`. Are the traces stitched
+(spans from both applications in the same trace) in Jaeger?
+
+Let's change the context propagation mechanism. In this application (not the
+other) let's add one more environment variable:
+
+- In the `docker/docker-compose.yml` file, change the `environment` section of the java service:
+```
+    environment:
+      ...
+      OTEL_PROPAGATORS: b3multi,baggage
```

Comment:
I don't know if we want to mess with `baggage` for the workshop. It might raise more questions than it answers at this point. I don't believe any instrumentation uses baggage for anything yet.

Reply:
Got it -- yeah, I was going to skip, but was not sure whether anything required it...